src/Core/View.js

```diff
@@ -458,4 +458,106 @@ View.prototype.execFrameRequesters = function execFrameRequesters(when, dt, upda
     }
 };
 
+function _raycastObjectsAt(raycaster, mouse, camera, object, target) {
+    // raycaster use NDC coordinate
+    const ndc = {
+        x: 2 * (mouse.x / camera.width) - 1,
+        y: -2 * (mouse.y / camera.height) + 1,
+    };
+    raycaster.setFromCamera(ndc, camera.camera3D);
+    const intersects = raycaster.intersectObject(object, true);
+    for (const inter of intersects) {
+        target.push(inter);
+    }
+}
+
+/**
+ * Return objects from some layers/objects3d under the mouse in this view.
+ *
+ * @param {Object} mouse - mouse position in window coordinates (0, 0 = top-left)
+ * @param {...*} where - where to look for objects. Can be either: empty (= look
+ * in all layers with type == 'geometry'), layer ids or layers or a mix of all
+ * the above.
+ * @return {Array} - an array of objects. Each element contains at least an object
+ * property which is the Object3D under the cursor. Then depending on the queried
+ * layer/source, there may be additionnal properties (coming from THREE.Raycaster
+ * for instance).
+ *
+ * @example
+ * view.pickObjectsAt({ x, y })
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding')
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding', myLayer)
+ */
+View.prototype.pickObjectsAt = function pickObjectsAt(mouse, ...where) {
```

Comment:
why `pickObjectsAt` isn't found in picking.js

Reply:
Because Picking.js is the implementation of the various picking methods (I'm also moving `raycastObjectsAt` in Picking.js)

`View.prototype.pickObjectsAt` is the boilerplate needed to use the pickings methods on a view.

Reply:
See previous answer.

It's boilerplate code to ease picking usage from the View.

Reply:
ok